template/source/layouts/header_footer_only.erb

```diff
@@ -1,91 +0,0 @@
-<!doctype html>
```

Comment:
Can you replace the contents in this file with `raise "Please remove me"`? Then we won't forget.